### ISSUE TITLE
Test failures when building with Emacs 27

### ISSUE THREAD

**Issue by Apteryks** [jorgenschaefer/elpy#1812]
# Summary
The following test fail when building with Emacs 27:

```
Warning [flymake test.py]: Disabling backend flymake-proc-legacy-flymake because (error Can’t find a suitable init function)
......................Test elpy-company-backend-should-add-shell-candidates backtrace:


Test elpy-company-backend-should-add-shell-candidates condition:

    (ert-test-failed
     ((should
       (string=
        (mapconcat ... cand " ")
        "variable_script"))
      :form
      (string= "" "variable_script")
      :value nil))

F.............................................................................Test elpy-fold-at-point-should-fold-and-unfold-comments backtrace:


Test elpy-fold-at-point-should-fold-and-unfold-comments condition:

    (ert-test-failed
     ((should
       (=
        (point)
        92))
      :form
      (= 104 92)
      :value nil))

F..........................................................................................................................................................................................................................................................................................................................................

Ran 431 tests in 50.556 seconds
2 unexpected results:
   FAILED  elpy-company-backend-should-add-shell-candidates
   FAILED  elpy-fold-at-point-should-fold-and-unfold-comments
command "ert-runner" failed with status 1
```
# Steps to reproduce
Running ert-runner with Emacs 27 (and the Elpy dependencies list below) in the sources of Elpy 1.34.0 triggers the problem.

# My configuration
## OS
GNU Guix, commit fb8542ed5c6b8d31bf09a8db0b9b14e8766c5dc5

## Result of `(elpy-config)`
```
Elpy Configuration

Emacs.............: 27.0.91
Elpy..............: 1.34.0
Virtualenv........: None
Interactive Python: python 3.8.2 (/gnu/store/jp9vm3sz2zm1wm99q793vvmjhcml1a4j-profile/bin/python)
RPC virtualenv....: rpc-venv (/home/maxim/.emacs.d/elpy/rpc-venv)
 Python...........: python 3.8.2 (/home/maxim/.emacs.d/elpy/rpc-venv/bin/python)
 Jedi.............: 0.17.0
 Rope.............: 0.11.0
 Autopep8.........: 1.3.5
 Yapf.............: 0.29.0
 Black............: 19.10b0
Syntax checker....: flake8 (/gnu/store/jp9vm3sz2zm1wm99q793vvmjhcml1a4j-profile/bin/flake8)
```

Note: The Virtualenv support is disabled because it is unnecessary when using Guix as a package manager.

## Elpy configuration in my init.el
None, the build happens in an isolated container.

**Comment by galaunay** [comment]
Thanks for the feedback.

I fixed the first one (`(run-python)` apparently now sets the python buffer as the current buffer).
The second one should have been fixed in a23626decd8e5a1a829d536072f8545c7b1d6f74.

**Comment by Apteryks** [comment]
I confirm this is fixed in 1.35.  Thank you! Closing.
